FILES/GetTattooShopDlcItemData.md

```diff
@@ -18,42 +20,28 @@ Character types:
 4 = MPFemale
 ```
 
-```csharp
-enum TattooZoneData
+```cpp
+struct sTattooShopItemData
 {
-    ZONE_TORSO = 0,
-    ZONE_HEAD = 1,
-    ZONE_LEFT_ARM = 2,
-    ZONE_RIGHT_ARM = 3,
-    ZONE_LEFT_LEG = 4,
-    ZONE_RIGHT_LEG = 5,
-    ZONE_UNKNOWN = 6,
-    ZONE_NONE = 7,
+	uint LockHash; // Hash that is to be used with IS_CONTENT_ITEM_LOCKED
+	int Index; // This is the unique ID of the tattoo, but sometimes can also be zero.
+	int Collection; // Which collection this tattoo belongs to
+	uint Preset; // Preset hash of this tattoo (AKA it's name, to be used with ADD_PED_DECORATION_FROM_HASHES)
+	int Cost; // How much this tattoo costs in shops
+	int eFacing; // Facing value of this tattoo
+	uint UpdateGroup; // Where exactly this tattoo is located on the ped. (E.G ARM_LEFT_FULL_SLEEVE.)
+	const char* NameTextLabel; // Name of this tattoo as a text label
 };
```

Comment:
Try to set a table with all this information.

Reply:
You can check https://github.com/citizenfx/natives/pull/984#issuecomment-1936900707